src/HealthChecks.AzureServiceBus/DependencyInjection/AzureServiceBusHealthCheckBuilderExtensions.cs

```diff
@@ -805,7 +805,7 @@ public static class AzureServiceBusHealthCheckBuilderExtensions
                 };
 
                 configure?.Invoke(options);
-                return new AzureServiceBusSubscriptionHealthCheck(options);
+                return new AzureServiceBusSubscriptionHealthCheck(options, new ServiceBusClientProvider());

```

Comment:
For all calls with `new ServiceBusClientProvider()` you may use something like default static instance.

Reply:
Or you may return ctor with 1 argument for backward compatibility. Then no changes required in this file.

Reply:
On it. For some reason I never excecuted the Api Approal Test. My bad.